src/main/resources/pages-config.yaml

```diff
@@ -125,9 +129,12 @@ pageDefinitions:
   - &landing
     name: landing
     pageTitle: landing.title
+    customPage: true
```

Comment:
I think I'm not fully understanding the benefit we get from explicitly specifying that this is a customPage, but I would love to learn more about the motivation behind this.

My first impression is that it would be nice if we could avoid having to specify explicitly that these static pages are custom pages. I really like how effortless it is to add static pages with nothing more than a `name` and `pageTitle` in our existing framework.

Are there static pages which we would want to specify `customPage: false` for? And what's different about those pages?

Reply:
Yeah, it defaults to false and thats how it knows to get a fragment "additionalContext"
So the intention is to be able to use the pages-config for "static" pages that look similar to "form" pages. 
Ex. even though this page uses a similar template as "formPage", we have to manually create this entire page like a c/p from form page and add all the content in it
https://github.com/codeforamerica/shiba/blob/main/src/main/resources/templates/emailDocsToYourCounty.html#L14-L46

With this refactoring, we can inject the header/footer/headerKey/contextFragment etc components for both static and form pages and just define the part that's different "additionalContext" for static pages.
<img width="394" alt="Screen Shot 2021-06-24 at 2 09 48 PM" src="https://user-images.githubusercontent.com/72890349/123333631-d8d7d980-d4f6-11eb-81c4-8a4ec6b09991.png">

The benefit is that the spacing around components will be consistent and we wouldnt have to update every static page if theres a change needed for that - It's less duplicate html.

"customPage: true" means the page doesnt conform to the existing template and has to be defined standalone the way "static" pages were before this refactoring

Reply:
Ah that's super helpful and super clear. Thank you for the writeup. I'm definitely onboard now and I can see the value of this customPage property.